src/rust/ensogl/lib/core/src/display/shape/primitive/def/primitive.rs

```diff
@@ -203,6 +203,28 @@ define_sdf_shapes! {
     }
 
 
+    // === RoundedLineSegment ===
+
+    /// A line segment from the origin to `target` with rounded endpoints.
+    RoundedLineSegment (target:Vector2<Pixels>, width:Pixels) {
```

Comment:
After working with it for a while, I think that the line segment should take a start point and an end point as arguments. That would more symmetric and would make chains of line segments much clearer. Would it be fine for you @wdanilo?

Reply:
Yes, it makes sesnee to me as well after longer thinking about it!